src/app/lib/config/services/arabic.js

```diff
@@ -198,6 +198,7 @@ export const service = {
         '/arabic/institutional/2011/01/000000_frequencies_radio',
       frequenciesPageLabel: 'استقبال البث',
       header: 'أحدث نشرة أخبار',
+      durationLabel: 'المدة %duration%',
```

Comment:
There is an issue for RTL services like Arabic that text and number get swapped places. In the config file duration is constructed as "text number", but in the DOM it is "number text"
<img width="601" alt="Screenshot 2020-05-07 at 10 10 59" src="https://user-images.githubusercontent.com/17802955/81276560-15264d00-904b-11ea-9a51-66a3b675ec69.png">

This is a problem only for RTL, does anybody know how this can be fixed?

Reply:
If you update the duration label in the config to be `"number text"`, will the element then be rendered as `"text number"`

Reply:
I have noticed then when there is RTL and LTR content mixed inside strings, text editors can display this content in a way that doesn't match how the page renders the content.

Reply:
Otherwise this looks good to me! Let me know when this question is resolved and I'll take another look?

Reply:
I think we should have the translation string in the order that means we get the correct output in the browser. I think the text editors we are using display in LTR when there is mixed content?

Reply:
After some investigation, I think I can say that it's fine as it is. It would probably nice to just get Sareh or someone else to check it using a screenreader but the main reasons I think it's fine is because in our config, it matches with everything else and in the DOM screenshot Olga posted, it is displaying `Duration (text) Time (number)` in the correct order if read RTL.
Some other interesting findings:
 - Webstorm changes the order of the 'words' so it is backwards RTL.
 - Sublime reverses the order of each RTL symbol and word so it becomes the arabic word backwards and then the number in LTR.
 - Chrome dev tools seems to support RTL and things are read RTL suggested by 
![image](https://user-images.githubusercontent.com/53564281/81311110-e9bc5600-907c-11ea-805a-1b1249a07ca0.png)
as 
![image](https://user-images.githubusercontent.com/53564281/81311188-fccf2600-907c-11ea-949d-ddb065ed19a3.png)